sound/soc/sof/sof-priv.h

```diff
@@ -363,6 +363,7 @@ struct snd_sof_dev {
 	struct snd_dma_buffer dmatp;
 	int dma_trace_pages;
 	wait_queue_head_t trace_sleep;
+	u32 host_init_offset; /* restore trace offset upon resume */
```

Comment:
is host_init_offset and host_offset both related to trace only ? If can we change the word host-> trace

Reply:
yes, they are both for trace only. let me change them.